recipes-core/iotedge-cli/iotedge-cli_1.1.3.bb

```diff
@@ -0,0 +1,277 @@
+    crate://crates.io/syn/1.0.11 \
+    crate://crates.io/synstructure/0.9.0 \
+    crate://crates.io/sysinfo/0.14.15 \
+    crate://crates.io/tabwriter/1.0.4 \
+    crate://crates.io/tempdir/0.3.7 \
+    crate://crates.io/tempfile/3.1.0 \
+    crate://crates.io/termcolor/0.3.6 \
+    crate://crates.io/termion/1.5.1 \
+    crate://crates.io/test-case/0.3.3 \
+    crate://crates.io/textwrap/0.9.0 \
+    crate://crates.io/thread_local/0.3.5 \
+    crate://crates.io/time/0.1.39 \
+    crate://crates.io/tokio-buf/0.1.1 \
+    crate://crates.io/tokio-codec/0.1.0 \
+    crate://crates.io/tokio-current-thread/0.1.6 \
+    crate://crates.io/tokio-executor/0.1.8 \
+    crate://crates.io/tokio-fs/0.1.6 \
+    crate://crates.io/tokio-io/0.1.8 \
+    crate://crates.io/tokio-reactor/0.1.1 \
+    crate://crates.io/tokio-signal/0.2.5 \
+    crate://crates.io/tokio-sync/0.1.7 \
+    crate://crates.io/tokio-tcp/0.1.0 \
+    crate://crates.io/tokio-threadpool/0.1.16 \
+    crate://crates.io/tokio-timer/0.2.11 \
+    crate://crates.io/tokio-tls/0.2.0 \
+    crate://crates.io/tokio-udp/0.1.0 \
+    crate://crates.io/tokio-uds/0.2.5 \
+    crate://crates.io/tokio/0.1.22 \
+    crate://crates.io/treediff/3.0.1 \
+    crate://crates.io/try-lock/0.2.2 \
+    crate://crates.io/typed-headers/0.1.1 \
+    crate://crates.io/typenum/1.10.0 \
+    crate://crates.io/ucd-util/0.1.1 \
+    crate://crates.io/unicase/2.1.0 \
+    crate://crates.io/unicode-bidi/0.3.4 \
+    crate://crates.io/unicode-normalization/0.1.5 \
+    crate://crates.io/unicode-width/0.1.4 \
+    crate://crates.io/unicode-xid/0.1.0 \
+    crate://crates.io/unicode-xid/0.2.0 \
+    crate://crates.io/unreachable/1.0.0 \
+    crate://crates.io/url/1.7.2 \
+    crate://crates.io/url/2.1.1 \
+    crate://crates.io/url_serde/0.2.0 \
+    crate://crates.io/utf8-ranges/1.0.0 \
+    crate://crates.io/vcpkg/0.2.3 \
+    crate://crates.io/vec_map/0.8.0 \
+    crate://crates.io/version_check/0.1.3 \
+    crate://crates.io/version_check/0.9.1 \
+    crate://crates.io/void/1.0.2 \
+    crate://crates.io/want/0.2.0 \
+    crate://crates.io/widestring/0.3.0 \
+    crate://crates.io/winapi-build/0.1.1 \
+    crate://crates.io/winapi-i686-pc-windows-gnu/0.4.0 \
+    crate://crates.io/winapi-x86_64-pc-windows-gnu/0.4.0 \
+    crate://crates.io/winapi/0.2.8 \
+    crate://crates.io/winapi/0.3.8 \
+    crate://crates.io/wincolor/0.1.6 \
+    crate://crates.io/windows-service/0.1.0 \
+    crate://crates.io/winreg/0.5.1 \
+    crate://crates.io/ws2_32-sys/0.2.1 \
+    crate://crates.io/yaml-rust/0.4.0 \
+    crate://crates.io/zip/0.5.3 \
+    git://github.com/Azure/hyperlocal-windows;protocol=https;nobranch=1;name=hyperlocal-windows;destsuffix=hyperlocal-windows \
+    git://github.com/Azure/mio-uds-windows.git;protocol=https;nobranch=1;name=mio-uds-windows;destsuffix=mio-uds-windows \
+    git://github.com/Azure/tokio-uds-windows.git;protocol=https;nobranch=1;name=tokio-uds-windows;destsuffix=tokio-uds-windows \
+"
+
```

Comment:
Could you please elaborate on what these variables are doing? 
Why are there `-windows` specify in yocto's stuff?

Reply:
I'm not sure - this is part of the generated code from cargo bitbake, and we still have Windows support in 1.1.  This is probably not actually useful, but I didn't want to alter the generated code.
I will flag the generated code vs the manual part so I don't have to answer these questions. :)